### ISSUE TITLE
Nastavit pravidelny inkrementalni update z SKC

### ISSUE THREAD

**Issue by JanMeritus** [NLCR/SeznamDNNT#307]
prosim o nastaveni

**Comment by JanMeritus** [comment]
prosim o doplneni vhodneho intervalu pro update od @Cypzirschova

**Comment by Cypzirschova** [comment]
Mělo by to být denně.

**Comment by smallcubes** [comment]
@pavel-stastny mohl byste mě prosím odkázat na příklad nastevní tohoto procesu nebo to zde prosím popsat

**Comment by pavel-stastny** [comment]
@smallcubes Ano, je to tak.  Nicméně kvůli issue #76 a #355 je nutné nejdříve provést operaci full

**Comment by smallcubes** [comment]
@pavel-stastny děkuji za potvrzení ve verzi 1.0.4.5 to tedy nastavím

**Comment by CIZKOVAL** [comment]
@smallcubes kdy bude update SKC probihat?

**Comment by smallcubes** [comment]
@CIZKOVAL zatím je to nastaveno na: každé pondělí ve 02:00

**Comment by JanMeritus** [comment]
@smallcubes pro účely testování tento týden prosim na omezenou dobu
nastavit na ta za 24h,  proč je ok

Dne po 25. 4. 2022 15:14 uživatel smallcubes ***@***.***>
napsal:

**Comment by smallcubes** [comment]
@JanMeritus moc dobře tomu nerozumím, chtěl bys nastavit ten inkrementální update, aby běžel každý den?

**Comment by smallcubes** [comment]
@pavel-stastny děkuji.

@JanMeritus nastaveno na: každý den od 05:00 `0 5 * * 0-6`, inkrementy většinou zaberou tak 15 minut
